Makefile

```diff
@@ -15,23 +15,27 @@ ifndef GOBIN
 endif
 
 BINARY=edgex-cli
-
+ 
 VERSION=$(shell cat ./VERSION 2>/dev/null || echo 0.0.0)
-GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-cli/cmd/version.Version=$(VERSION)"
+TIME=$(shell date)
+GOFLAGS=-ldflags "-X 'github.com/edgexfoundry/edgex-cli.BuildVersion=$(VERSION)' -X 'github.com/edgexfoundry/edgex-cli.BuildTime=$(TIME)'"
 ARTIFACT_ROOT?=bin
 
 build:
 	@echo "GOPATH=$(GOPATH)"
-	$(GO) build -o $(BINARY) $(GOFLAGS)
+	$(GO) build -o ${ARTIFACT_ROOT}/$(BINARY) $(GOFLAGS) ./cmd/edgex
+
+tidy:
+	go mod tidy
 
 # initial impl. Feel free to override. Please keep ARTIFACT_ROOT coming from env though. CI/CD pipeline relies on this
 build-all:
 	@echo "GOPATH=$(GOPATH)"
 
-	GOOS=linux GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY)-linux-amd64 $(GOFLAGS)
-	GOOS=linux GOARCH=arm64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY)-linux-arm64 $(GOFLAGS)
-	GOOS=darwin GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY)-mac $(GOFLAGS)
-	GOOS=windows GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY).exe $(GOFLAGS)
+	GOOS=linux GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY)-linux-amd64 $(GOFLAGS) ./cmd/edgex
```

Comment:
binary is `edgex`, not `edgex-cli`

Reply:
BINARY is set to `edgex-cli` in line 17. However, this refers to the directory containing `/cmd/edgex/main.go`. That directory should really be named `edgex-cli` as well for consistency, so I've renamed it and updated this accordingly. Everything should now therefore use `edgex-cli`